src/gtk/toga_gtk/widgets/numberinput.py

```diff
@@ -16,12 +16,24 @@ def create(self):
 
         self.rehint()
 
+    def set_readonly(self, value):
+        self.native.editable = not value
+
     def get_value(self):
         return self.native.get_value()
 
     def set_value(self, value):
         self.native.set_value(value)
 
+    def set_alignment(self, value):
```

Comment:
Seeing this boilerplate having to be copied-and-pasted to so many places, and having it be effectively `None` right now, smells funny, but it's not a blocker.

Reply:
That's a fair cop. There's an interesting gap here between "the test fails if the method doesn't exist", and "we know the test won't work because it isn't implemented". We possibly need to do a thing similar to the `not_implmeneted` list in Batavia/VOC so we can flag areas that need work, but have been stubbed out for API completeness.